src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1579,14 +1595,17 @@ export class PostgresStorageAdapter {
   }
 
   performInitialization({ VolatileClassesSchemas }) {
+    // TODO: This method needs to be rewritten to make proper use of connections (@vitaly-t)
```

Comment:
Regarding this TODO we'll want to at least rework this in with transactions as mentioned below.

Reply:
I will do it :wink:

Reply:
DONE!

Reply:
P.S. As for the last item, the `TODO` thing, it definitely should done as a separate PR.

Reply:
Noted! @paulovitin you're good for that one then, we'll address that down the road.